src/ts/src/common/types/table.d.ts

```diff
@@ -1,5 +1,18 @@
 interface Header {
     [index: string]: string;
     text: string;
-    key: keyof BioinformaticsType;
+    key: string | number;
```

Comment:
shouldn't this just always be a number?

Reply:
`key` here refers to the keys of a JS object, like on a sample or tree, which will almost always be strings.

Reply:
Should we just restrict the type of `key` to be string?

The current use cases seem to be string anyway 😆 

```ts
export const SAMPLE_HEADERS: Header[] = [
  {
    key: "privateId",
    text: "Private ID",
  },
  {
    key: "publicId",
    text: "Public ID",
  },
  {
    key: "uploadDate",
    text: "Upload Date",
  },
  {
    key: "collectionDate",
    text: "Collection Date",
  },
  {
    key: "collectionLocation",
    text: "Collection Location",
  },
  {
    key: "gisaid",
    text: "GISAID",
  },
];

export const TREE_HEADERS: Header[] = [
  {
    key: "name",
    text: "Tree Name",
  },
  {
    key: "creationDate",
    text: "Creation Date",
  },
  {
    key: "pathogenGenomeCount",
    text: "Total Samples",
  },
  {
    key: "downloadLink",
    text: "",
  },
];

```